pkg/hubble/parser/seven/http.go

```diff
@@ -7,15 +7,26 @@ import (
 	"fmt"
 	"net/url"
 	"sort"
+	"strings"
 	"time"
 
+	"golang.org/x/exp/slices"
+
 	flowpb "github.com/cilium/cilium/api/v1/flow"
 	"github.com/cilium/cilium/pkg/hubble/defaults"
 	"github.com/cilium/cilium/pkg/hubble/parser/options"
 	"github.com/cilium/cilium/pkg/proxy/accesslog"
 )
 
 func decodeHTTP(flowType accesslog.FlowType, http *accesslog.LogRecordHTTP, opts *options.Options) *flowpb.Layer7_Http {
+	allowedHeaders := []string{
```

Comment:
Can you make it a package constant instead of a function-scoped var? Unless there is a better place to define it.